package.json

```diff
@@ -28,11 +29,9 @@
     "gulp-connect": "~2.0.5",
     "gulp-html2string": "git://github.com/Rise-Vision/gulp-html2string",
     "gulp-minify-css": "^0.3.5",
-    "gulp-protractor": "~0.0.8",
     "gulp-rename": "~1.2.0",
     "gulp-sass": "^0.7.2",
     "gulp-util": "~2.2.17",
-    "mocha": "~1.20.1",
     "phantomjs": "^1.9.7",
     "protractor": "~1.0.0",
```

Comment:
@donnapep You should remove the dependency on protractor and rely on widget-tester, thanks